skripchenkoveta/hometask-4/spec/my_hash_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require '../my_hash'
+require 'rspec'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe MyHash do
+  subject(:my_hash) { described_class.new }
+
+  before do
+    my_hash['li'] = 'za'
+    my_hash['my'] = 13
+    my_hash[:b] = 'name'
+    my_hash[404] = 'error'
+  end
+
+  describe '#[]=' do
+    context 'introduction of new elements key=>value' do
```

Comment:
usually context description starts with `when...`
but here you just basically call the method `[]=` and you expect this to return the value
so `context` is redundant here, because it's just a regular scenario of using the method
so it would be
```
describe '#[]=' do
  it 'returns the value' do
    expect(my_hash[:c] = 10).to eq 10
    expect(my_hash[1] = -3).to eq(-3)
  end
```